bw_simapro_csv/blocks/process.py

```diff
@@ -0,0 +1,166 @@
+import re
+from typing import List
+
+from loguru import logger
+
+from ..utils import asboolean, asdate
+from .base import SimaProCSVBlock
+
+LIST_ELEMENTS = {
+    "Avoided products",
+    "Economic issues",
+    "Electricity/heat",
+    "Emissions to air",
+    "Emissions to soil",
+    "Emissions to water",
+    "Final waste flows",
+    "Materials/fuels",
+    "Non material emissions",
+    "Products",
+    "Resources",
+    "Social issues",
+    "Waste to treatment",
+    "Waste treatment",
+}
+HAS_SUBCATEGORY = {
+    "Economic issues"
+    "Emissions to air",
+    "Emissions to soil",
+    "Emissions to water",
+    "Final waste flows",
+    "Non material emissions",
+    "Resources",
+    "Social issues",
+}
+NO_SUBCATEGORY = {
+    "Avoided products",    
+    "Electricity/heat",
+    "Materials/fuels",
+    "Products",
+    "Waste to treatment",
+}
+PARAMETERS = {
+    "Calculated parameters",
+    "Input parameters",
+}
+
+
+# Exclude `e` for exponent
+has_letters = re.compile("[a-df-zA-CF-Z]+")
```

Comment:
This field (it is unlabelled) can be either a floating point number or a text string which should be interpreted as a formula. I know this is hacky, and maybe misses `NaN` (this would be fine, it would be interpreted correctly by the formula parser), but I don't have a great alternative - especially as sometimes the unit and "amount" fields have their order switched...